extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -28,22 +32,34 @@ class AmpAccordion extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    /** @const @private {boolean} */
+    this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
```

Comment:
It's because we want to guard this property before validator provide support for this attribute.